Integrations/PolySwarm/Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "*"
+mock = "*"
+pytest-mock = "*"
+requests = "*"
+requests-mock = "*"
+
+[packages]
+pylint = "*"
+pytest = "*"
+mock = "*"
+pytest-mock = "*"
+requests = "*"
+requests-mock = "*"
+builtins = "*"
+
+[requires]
+python_version = "2.7"
```

Comment:
seems that you are using python3 but in Pipfile it defined 2.7.
please update the python version

Reply:
i'm using python2.7